_dev/global.d.ts

```diff
@@ -29,6 +29,7 @@ interface AutoUpgradeVariables {
   ps_version: string;
   bo_language: string;
   bo_timezone: string;
+  help_link: string;
```

Comment:
I usually end by adding several links to the object visible on the visible, and to avoid a use list of properties when getting the root of this variable, I move it in a sub key `links`

```suggestion
  links: {
      help: string,
  };
```

Reply:
good idea